organizations/v0/urls.py

```diff
@@ -0,0 +1,18 @@
+"""
+URLS for organizations end points.
+"""
+from django.conf.urls import patterns
+from rest_framework_extensions.routers import ExtendedSimpleRouter
+
+from organizations.v0.views import OrganizationsViewSet
+
+
+# pylint: disable=invalid-name
+urlpatterns = patterns(
+    '',
+)
+
+router = ExtendedSimpleRouter()
```

Comment:
Why are you using this router rather than `SimpleRouter`? You should not need DRF-extensions at all.